package.json

```diff
@@ -8,11 +8,13 @@
   },
   "scripts": {
     "algolia": "node ./algolia.js",
+    "pre-build": "npm-run-all clean ignore gulp rollup maybe-sync-external",
```

Comment:
I found using the `pre-` prefix here a little confusing, because `pre` (without the `-`) and `post` have a special meaning in `npm` scripts: https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

Given that you're not using the special `pre` behavior, could you name this something different, so that others aren't confused? Like `setup` or something similar?